minio/minio
# Don't exit background append if backend specific files show up

As part of background append Minio expects part files to be of the form `partNumber.etag.size`. In some backend filesystems, like in #7518, special files appear in Minio's internal upload directory, causing background append to exit due to invalid part file name. We should skip non part files.

## Description

The PR ignores invalid part file name error to accommodate backend specific files appearing during a background append.
Fixes #7518 
## Motivation and Context


See #7518 

## Regression


No
## How Has This Been Tested?



1. Start a multipart upload
2. Add a file like `/etc/issue` to the upload directory in Minio's backend
3. Complete the multipart upload and check if file is successfully uploaded

## Types of changes


## Checklist:
